specs/PostalCode.spec.ts

```diff
@@ -13,6 +13,12 @@ describe('PostalCode', () => {
         expect(postalCode.format('AD')).toBe('AD-700');
     });
 
+    it('does not format a postal code that is not valid for a country', () => {
```

Comment:
Just a question what is the purpose of the PostalCode if it allow invalid values.
should it be then just a string instead?

Reply:
It does not allow invalid values. The context of the country is, however, not known upfront always (and can change in the process), hence the extra restriction.